tests/testthat/test-config-log.R

```diff
@@ -211,3 +211,61 @@ test_that("add_config() works no json found [BBR-CGLG-013]", {
   expect_equal(nrow(log_df), 1)
   expect_true(all(c(ABS_MOD_PATH, RUN_ID_COL, YAML_TAGS) %in% names(log_df)))
 })
+
+# ##########################################
+# # Testing Additional Parameters Passed
+# ##########################################
+
+
+test_that("config_log() works with filtering parameter numeric [BBR-CGLG-014]",
+          {
+            setup_this_test <- function() {
+              create_rlg_models()
+
+              purrr::walk(
+               c(2, 3),
+               ~fs::dir_copy("{MODEL_DIR}/1/" %>% glue(), "{MODEL_DIR}/{.x}/" %>% glue())
```

Comment:
I think it'd be good to use `file.path()`, as is already done elsewhere.

Reply:
I made sure update this.  Thanks